unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/PrimitiveMesh/PrimitiveMeshFactory.cs

```diff
@@ -29,7 +29,7 @@ public static Mesh CreateMesh(AssetPromise_PrimitiveMesh_Model meshModelModel)
                 }
                 else
                 {
-                    mesh = PrimitiveMeshBuilder.BuildSphere(1f);
```

Comment:
Maybe we can take the opportunity to extract a constant, so next time we only have to change it in one place.

Reply:
You made me realize that I should have removed that line 😅
Actually default value it's already defaulted in the sphere property constructor here:
https://github.com/decentraland/unity-renderer/blob/1e96f9fc66e0c81efdc703cdb9b96c32f92a88a0/unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/PrimitiveMesh/AssetPromise_PrimitiveMesh_Model.cs#L81